tools/example_scripts/deletemincsqlwrapper.pl

```diff
@@ -0,0 +1,116 @@
+#!/usr/bin/perl
+use strict;
+use warnings;
+use Getopt::Tabular;
+no warnings 'once';
+use Data::Dumper;
+use File::Basename;
+use File::Copy;
+use Term::ANSIColor qw(:constants);
+use NeuroDB::DBI;
+
+my $profile = undef;
+my $insertminc;
+
+my @opt_table           = (
+    [ "Basic options", "section" ],
+    [
+        "-profile", "string", 1, \$profile,
+        "name of config file in ../dicom-archive/.loris_mri"
+    ],
+    [   
+         "-insertminc", "boolean", 0, \$insertminc, "Re-insert the deleted minc"
+    ]
+);
+
+my $Help = <<HELP;
+******************************************************************************
+Wrapper to minc_deletion.pl for bulk deletion based on a SQL customisable Query
+******************************************************************************
+
+This script is a wrapper for deleting multiple mincs at a time and optionally 
+re-inserting them.
+
+It will pause before for confirmation before deleting.
+
```

Comment:
Could be good to add a reference to the README section so that if people wants to have a better idea of what the script does and how to use it, they can refer to it.

Reply:
added a line there.

Reply:
good point, done